lib/vanagon/driver.rb

```diff
@@ -116,14 +116,21 @@ def install_build_dependencies # rubocop:disable Metrics/AbcSize
       end
     end
 
-    def run # rubocop:disable Metrics/AbcSize
+    def run # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
       # Simple sanity check for the project
       if @project.version.nil? or @project.version.empty?
         raise Vanagon::Error, "Project requires a version set, all is lost."
       end
 
-      @engine.startup(workdir)
+      # if only_fetch has been set in the project, don't execute the
+      # whole makefile. Instead just perform the installation.
+      if @project.only_fetch.empty?
```

Comment:
Minor. Would rather see this as a guardian clause:

```ruby
make_target = ''
if ! @project.only_fetch.empty?
  make_target = @project.name + '-project'
end
```

Reply:
I know this has already been updated, but consider `unless` instead if `if !` -- super minor stylistic choice but also very in-line with the overall code throughout Vanagon